packages/elements/src/dialog/index.ts

```diff
@@ -344,8 +396,8 @@ export class Dialog extends Overlay {
    */
   protected get headerRegion (): TemplateResult {
     return html`
-      ${this.header === null ? this.t('HEADER') : this.header}
-      <ef-icon part="close" icon="cross" slot="right" @tap="${this.defaultCancel}"></ef-icon>
+      <span aria-hidden="true">${this.header === null ? this.t('HEADER') : this.header}</span>
```

Comment:
Why is this hidden ?

Reply:
To prevent the screen reader to announce the header again after it read from `aria-label`